src/.nuget/toolchain.ubuntu.14.04-arm64.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>toolchain.ubuntu.14.04-arm64.Microsoft.DotNet.ILCompiler.Development</id>
```

Comment:
Please do co-ordinate with @ericstj to get the changes to Microsoft.NETCore.Platforms in before merging this. Otherwise you'll see a lot of things fail weirdly.